Source/WebCore/rendering/svg/SVGRenderingContext.cpp

```diff
@@ -96,7 +96,7 @@ void SVGRenderingContext::prepareToRenderSVGContent(RenderElement& renderer, Pai
     // Setup transparency layers before setting up SVG resources!
     bool isRenderingMask = isRenderingMaskImage(*m_renderer);
     // RenderLayer takes care of root opacity.
-    float opacity = (renderer.isLegacyRenderSVGRoot() || isRenderingMask) ? 1 : style.opacity();
+    float opacity = (renderer.isRenderOrLegacyRenderSVGRoot() || isRenderingMask) ? 1 : style.opacity();
```

Comment:
This affects the testcase on LBSE only?
If so, then, the test should be run with LBSE on.